src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -170,10 +164,8 @@ private String extractRequestBody(StaplerRequest req) {
         return body;
     }
 
-    
-    private Set<GhprbWebHook> getWebHooks() {
-        final Set<GhprbWebHook> webHooks = new HashSet<GhprbWebHook>();
-
+    private Set<GhprbTrigger> getTriggers(String repoName, String body, String signature) {
+        final Set<GhprbTrigger> triggers = new HashSet<GhprbTrigger>(5);
```

Comment:
This is a list to be sent back of matching triggers for a repo.  Would it be better to keep a static list of repo's and which triggers are pointing at them?  Then we wouldn't need to loop through every job.